### DISCUSSION TITLE
How do i delete old slash commands?

### DISCUSSION THREAD

**Discussion by REALWennix** [discord-jda/JDA#2347] [Questions and Help]
Hi
Im having issue with deleting slash commands that i dont need. 
I tried deleteCommandById() but that didnt worked.
Any idea how can i delete specific slash commands?

Thanks.

![image](https://user-images.githubusercontent.com/82824312/205466735-6923f814-2da6-47dd-919e-56a60262f317.png)

**Comment by michel929** [comment]
Hey,
U used `upsertCommand()` or `updateCommands().addCommands(allOfYourCommands).queue()` ?

In my Projects i use `upsertCommand()`. This will replace the existing command and make the client cache invalid.
